### PR TITLE
Improve #53 - Added filter_var instead of direct checking

### PR DESCRIPTION
**Pull request by partikus**
It's allow to use sitemap option in the xml file. 
Please let me know if any other test cases are needed. 
@alain-flaus code review pls

### CODE REVIEW

**Comment by partikus** [comment]
@alain-flaus ping

**Comment by partikus** [comment]
Please take a look on that, thx 
 @esion @alain-flaus @kostiklv @Koc

File: Tests/EventListener/RouteAnnotationEventListenerTest.php [hunk 1]
```diff
@@ -34,7 +34,7 @@ public function testNoAnnotation()
      */
     public function testInvalidSitemapFalse()
     {
-        $this->setExpectedException('InvalidArgumentException');
+        $this->setExpectedException('\InvalidArgumentException');
```
Comment (Koc) [inline Tests/EventListener/RouteAnnotationEventListenerTest.php]: IMHO there is no sense for backslash here

Comment (partikus) [inline Tests/EventListener/RouteAnnotationEventListenerTest.php]: I propose to unify all usages of ```InvalidArgumentException``` in the file. 
We can use ```InvalidArgumentException``` for each usage(without backslash). 
@Koc What do you think?

Comment (Koc) [inline Tests/EventListener/RouteAnnotationEventListenerTest.php]: Better remove backslashes when using class name as string. No sense for it.

**Comment by partikus** [comment]
@Koc done and fixed

**Comment by partikus** [comment]
ping @Koc @alain-flaus

**Pull request merged**